auth.go

```diff
@@ -243,12 +278,15 @@ type Central struct {
 	NewSessionExpiresAfter time.Duration
 	DisablePasswordReuse   bool
 	PasswordExpiresAfter   time.Duration
+	MaxFailedLoginAttempts int // only applies if EnableAccountLocking is true
```

Comment:
Can't we move MaxFailedLoginAttempts out into the LockingPolicy?
And perhaps same for EnableAccountLocking?